ops/utils/gauntlet.go

```diff
@@ -84,7 +84,7 @@ func (g Gauntlet) ExecCommand(arg ...string) error {
 	a := []string{}
 	a = append(a, g.arg...)
 	a = append(a, arg...)
-	// Execute
+	// #nosec - Execute
```

Comment:
I don't recognize this format. Is there an advantage over `//nolint`? https://golangci-lint.run/usage/false-positives/#nolint-directive
```suggestion
	//nolint:nosec - Execute
```

Reply:
oh I didn't know golangci-lint had it's own - the #nosec is just directly interacting with the underlying linter. let me give it a shot

Reply:
gave it a run, but `nolint` doesn't do anything?

Reply:
:shrug:

Reply:
Weird:
https://github.com/golangci/golangci-lint/blob/master/pkg/result/processors/testdata/nolint2.go#L9
https://github.com/golangci/golangci-lint/blob/master/pkg/result/processors/testdata/nolint_bad_names.go#L13

Reply:
I wonder if `nolint` is for other types of linters? and `#nosec` is specific? https://github.com/golangci/golangci-lint/blob/1948081d84d96254f3da9d409d185e0942f1132b/.golangci.reference.yml#L888